pages/community/rse-groups.md

```diff
@@ -1,15 +1,12 @@
 ---
 layout: page
 title: Organizations with RSEs
-subtitle:
-bigimg: 
+subtitle: The following institutions have joined the US-RSE community
```

Comment:
I'm not comfortable with this wording - I don't think we have a well-defined concept of an institution joining the community.  @cosden, what do you think?

Reply:
We could reword, or just remove entirely. The page is sort of self explanatory :)

Reply:
I didn't have a chance to look at anything other than this comment (and probably won't today - sorry)  I think @cferenba is right we shouldn't imply an institution joined.  The list is also clearly not exhaustive, lots of people have joined and haven't put their affiliation there.  I'd go with removing it, leaving it as is "RSEs from the following institutions and organizations have joined the US-RSE community" or have something that says this is just a subset of the institutions who have members in the US-RSE.

Reply:
@cosden would you be ok with removing the subtitle entirely? The page speaks for itself, and I did try the original (longer) version and it looked a bit wacky. if you want to keep it, what I can do is move it back to be a centered paragraph (so it just takes one line). I think once I fix that, @cferenba is good with the rest so we are good to merge.